src/MDMTransitionController.h

```diff
@@ -44,5 +45,5 @@ NS_SWIFT_NAME(TransitionController)
  This may be non-nil while a transition is active.
  */
 @property(nonatomic, strong, nullable, readonly) id<MDMTransition> activeTransition;
-
+@property(nonatomic, strong, nullable) id<MDMInteractiveTransition> interactiveTransition;
```

Comment:
This technically isn't a "transition" as much as an interaction controller for a transition. Can we consider renaming this to `transitionInteractionController`?